tests/language-feature/capability/var-capability-wgsl-2.slang

```diff
@@ -0,0 +1,17 @@
+//TEST:SIMPLE(filecheck=CHECK): -target wgsl -entry Main -stage compute
+//TEST:SIMPLE(filecheck=CHECK_IGNORE_CAPS): -target wgsl -entry Main -stage compute -ignore-capabilities
+
+//CHECK:error 36108: 'input' has dependencies that are not compatible on the required target 'wgsl'.
```

Comment:
I don't think this should result in an error, because `Main` doesn't reference `inputWrapper` at all, so it shouldn't be considered part of program when compiling `Main`. You will need to reference `input` from `Main` to trigger an error.